pkl-gradle/src/main/java/org/pkl/gradle/task/EvalTask.java

```diff
@@ -19,32 +19,26 @@
 import org.gradle.api.file.DirectoryProperty;
 import org.gradle.api.file.RegularFileProperty;
 import org.gradle.api.provider.Property;
-import org.gradle.api.tasks.Input;
-import org.gradle.api.tasks.Optional;
-import org.gradle.api.tasks.OutputDirectory;
-import org.gradle.api.tasks.OutputFile;
+import org.gradle.api.tasks.Internal;
+import org.gradle.api.tasks.UntrackedTask;
 import org.pkl.cli.CliEvaluator;
 import org.pkl.cli.CliEvaluatorOptions;
 
+@UntrackedTask(because = "Output file names are known only after execution")
```

Comment:
Maybe explain it a bit better here as well (that the output files contain placeholders)? 🤔

Reply:
Done!